engine/samples/gizmos/main.cpp

```diff
@@ -0,0 +1,56 @@
+#include <cubos/engine/cubos.hpp>
+#include <cubos/engine/gizmos/plugin.hpp>
+#include <cubos/engine/renderer/plugin.hpp>
+#include <cubos/engine/transform/plugin.hpp>
+
+using cubos::core::ecs::Commands;
+using cubos::core::ecs::Write;
+
+using cubos::engine::Gizmos;
+
+using namespace cubos::engine;
+
+static void mockCamera(Write<ActiveCameras> camera, Commands cmds)
+{
+    camera->entities[0] = cmds.create()
+                              .add(Camera{.fovY = 60.0F, .zNear = 0.1F, .zFar = 100.0F})
+                              .add(LocalToWorld{})
+                              .add(Position{{5.0F, 5.0F, -10.0F}})
+                              .add(Rotation{glm::quatLookAt(glm::vec3{0.0F, 0.0F, 1.0F}, glm::vec3{0.0F, 1.0F, 0.0F})})
+                              .entity();
+}
```

Comment:
Why do we call this a mock camera?

Reply:
Because i copy pasted it from somewhere and forgot to change the name :)